About.html

```diff
@@ -0,0 +1,182 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone ProjectAbout</title>
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/d6cd3c7604.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav class="navBar">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="nav-container">
+              <div class="nav-linkA"></div>
+                <div>
+                <ul class="nav-link">
+                    <li>
+                        <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+                    </li>
+                    <li>
+                        <a href=""><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li>
+                        <a href="">English</a>
+                    </li>
+                    <li>
+                        <a href="#Home-page">My Page</a>
+                    </li>
+                    <li>
+                        <a href="">Log out</a>
+                    </li>
+                </ul>
+            </div>
+            </div>
+                <div class="nav-branding">
+                    <div class="nav-logo">
+                        <a href="">
+                          <img
+                            id="img-header"
+                            src="./images/IMG-header-removebg-preview.png"
+                            alt=""
+                          />
+                        </a>
+                      </div>
+                      <div class="nav-link-container">
+                <ul class="nav-links">
+                    <li>
+                        <a id="about" href="./index.html" class="link">About</a>
+                    </li>
+                    <li>
+                        <a href="" class="link">Program</a>
+                    </li>
+                    <li>
+                        <a href="" class="link">Join</a>
+                    </li>
+                    <li>
+                        <a href="" class="link">join</a>
+                    </li>
+                    <li>
+                        <a href="" class="link">Sponsors</a>
+                    </li>
+                    <li>
+                        <a href="./About.html" class="link">News</a>
+                    </li>
+                    <li>
+                        <a id="nav-link-lastchild" href="" class="link">YEHO WEHD</a>
+                    </li>
+                </ul>
+            </div>
+            </div>
+        </nav>
+    </header>
+    <main class="Home-page">
+        <section class="home-cont">
+        <div class="home-text">
+        <p>Hello Sharing World!</p>
+        <h1>WORLD <br>ENVIRONMENTAL<br>HEALTH DAY 2022</h1>
+        </div>
```

Comment:
- [ ] On mobile screen there are multiple issues present. For example, the text is not aligned. 

![image](https://user-images.githubusercontent.com/58504655/177001842-38b20ac9-1cdb-4bac-a5fd-e8962993dc3c.png)